exe-common/src/network/api.rs

```diff
@@ -30,7 +36,7 @@ pub trait Api {
         got_block: &mut F,
     ) -> Result<()>
     where
-        F: FnMut(&HeaderHash, &Block, &RawBlock) -> ();
+        F: FnMut(&HeaderHash, &Block, &RawBlock) -> BlockReceivingFlag;
```

Comment:
This one will break any other users. The only alternative, I guess, is to add an alternative `get_blocks` function with a default implementation. Do we want that?